addon/components/ember-times-header.hbs

```diff
@@ -1,3 +1,4 @@
+{{! template-lint-disable no-unknown-arguments-for-builtin-components }}
 {{#if this.isNewsletter}}
     <LinkTo @route="tag" @model="Newsletter" class="blog-header" @aria-label="Read more issues of The Ember Times">
```

Comment:
Question- do you need `@aria-label` or can you just use `aria-label`?

Reply:
so this isn't something that's changed as part of this PR, What we should do is get this merged and then fix as many issues that are ignored using lint-to-the-future after this is merged so we don't let this PR spiral any further out of control 🙈